spring-amqp/src/test/java/org/springframework/amqp/support/converter/Jackson2JsonMessageConverterTests.java

```diff
@@ -478,6 +490,31 @@ public List<Fiz> fizLister() {
 		return null;
 	}
 
+	public static class JodasRevenge {
```

Comment:
I'm not sure that this is OK name for professional code.
Something like `TestJsonData` should be enough.
Doesn't look like this class really performs some revenge from Master Joda 😉 .

Plus it feels like this class could be just a `record`.